.github/workflows/build-containers.yml

```diff
@@ -0,0 +1,69 @@
+name: Build SPRAS Containers
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-remove-omics1:
+    uses: "./.github/workflows/build-and-remove-template.yml"
+    with:
+      path: docker-wrappers/OmicsIntegrator1
+      container: reedcompbio/omics-integrator-1
+      tag: latest
+  build-and-remove-omics2:
+    uses: "./.github/workflows/build-and-remove-template.yml"
+    with:
+      path: docker-wrappers/OmicsIntegrator2
+      container: reedcompbio/omics-integrator-2
+      tag: latest
+  build-and-remove-pathlinker:
+    uses: "./.github/workflows/build-and-remove-template.yml"
+    with:
+      path: docker-wrappers/PathLinker
+      container: reedcompbio/pathlinker
+      tag: v2
```

Comment:
I haven't been very thoughtful about which tags to test. Should we go back to testing latest for all of these?

Reply:
These tags are arbitrary -- they're set for the image that gets created in the action, but the image is immediately deleted and never pushed because all we're testing is that it builds. We could set the tags to anything we want! After reflecting a bit, I think I'll just remove the tag field as an input and always set things to latest.